Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Please double check the PropertyGroup format from guidelines https://github.com/FrendsPlatform/FrendsTasks/wiki#task-checklist. You can also look at other tasks how the PropertyGroup should be constructed.